app/build.gradle

```
@@ -23,6 +23,15 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
         }
     }
+
+    dataBinding {
+        enabled true
+    }
+
+    viewBinding {
+        enabled true
+    }
+
```

```suggestion
    buildFeatures {
        viewBinding true
        dataBinding true
    }
```

아마 공식가이드에서는 [`buildFeatures` 사용](https://developer.android.com/topic/libraries/view-binding#groovy) 권장하고 있을거에요

---

넵 감사합니다